docs/publishers/cli.md

```diff
@@ -0,0 +1,65 @@
+# DPM: the datapackage manger CLI
+
+The dpm is a command-line tool aimed to help publishers to prepare and upload data to the Datapackage Registry. With dpm you will be able to:
+
+* Validate your data to ensure its quality
+* Upload datapackage
+* Tag uploaded datapackage to create historical snapshot
+* Remove uploaded datapackage that is no longer needed
+
+## Installation
+
+Dpm is available on pypi [hopefully soon it will be], you can install it using pip in the terminal:
+```
+[sudo] pip install dpm
+```
+
+**NOTE**
+
+You can alternatively install unstable version directly from the code repository:
+
+```
+[sudo] pip install git+https://github.com/frictionlessdata/dpm-py.git
+```
+
+## Configuration
+
+Dpmpy can be configured using `dpmpy configure` command. It will ask you
+to provide username, access_token and server address of datapackage registry.
+
+The config is stored in `~/.dpm/config`, you can edit it with text editor.
+Simple example config file can look like this:
+
+```
+username = myname
+access_token = mykey
+server = https://staging.datapackaged.com
+```
+
+## Usage
+
+### Publish
+
+To publish datapackage, go to the datapackage directory (with `datapackage.json`) and
+launch `dpmpy publish`. If your configured *username* and *access_token* are correct, dpmpy will
+upload datapackage.json and all relevant resources to the registry server.
+
+[TODO] add details in the [upload](upload/) section
```

Comment:
we can expand on what options are available (skipping datavalidation for ex), what exeacly the command does in what order, and what you can expect to happen on the server.